### ISSUE TITLE
SEO Debrief - Account Benefits Page

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#6348]
| Estimates |  |
| ------------- | ------------- | 
| TL | 2|
| PM | 0|
| ENG | 2|
| SEO | 5|
| ANA | 0|
| DES | 2|
| COPY |0| 

Description, good enough that reading the card 1 month from now, you still know what it means.

---

#### Success Criteria 💛 

- [x] Raphael is finally up to speed
- [ ] We have sufficient SEO data to decide whether
- [ ] How does this fit in RR's Fx features project
- [x] Set of SEO keywords to rank for

**Comment by ejregithub** [comment]
@dzingeek to consult w. @rraue today

**Comment by rraue** [comment]
We should unite firefox/accounts/, firefox/accounts/features/ and firefox/features/sync/ to one page on /firefox/accounts/. The content and function for our website users are too similar. The new page should have a clear cta (different for fx and non-fx users only in cta, not in content) and copy should be about all benefits of accounts/sync. 

- [ ] If we still need /features or /sync for campaigning, the canonical of both pages should point to /accounts (locale sensitive).

- [ ] If we do not need these two pages, they should be redirected 301 to /accounts (locale sensitive)

**Keywords**
Interesting is, that "Firefox sync" is still a bigger search term than "firefox accounts". We consider this and clear functions and branding. That is one reason, why it is better to have one page explaining it all, than three different pages like now.

Here are the most important keywords, we are ranking for right now, with all three pages: https://docs.google.com/spreadsheets/d/1MBSE3nccBOreylhy4amJ3V6jB3bEDyDzSRsXxNa9hII/edit?usp=sharing

tldr: we need to make sure that all feautures are described and listed and as well we need to make the connection clear of "Firefox", "Mozilla", "Accounts", "Sync" and "Login".

**Link structure**

- [ ] The new /accounts/ page should be linked in the main navigation, from /firefox/ and /firefox/mobile/. 

- [ ] All links on blog.mozilla.org to the above mentioned three pages should be checked and link to /accounts/ to be redirected. (TBD)

**Comment by ejregithub** [comment]
@djst - @rraue provided his assessment/advice in the comment above. We implemented the first item but did not, AFAIK, pursue the other two in the haste to take the page live.  I'm, moving this to the Post MVP column for now. Could you please review and prioritize the other two items, letting me know if we should split these out to two new separate issues to add to the backlog?
